helpers/movimientosArkaHelper/movimientosArkaHelper.go

```diff
@@ -43,37 +42,70 @@ func GetAllEstadoMovimiento(nombre string) (estado []*models.EstadoMovimiento, o
 	return resEstadoMovimiento, nil
 }
 
-func GetAllElementosMovimiento(query string) (estado []*models.ElementosMovimiento, outputError map[string]interface{}) {
+func GetAllElementosMovimiento(query string) (elementos []*models.ElementosMovimiento, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetAllElementosMovimiento hace tal cosa"
```suggestion
// GetAllElementosMovimiento ...
func GetAllElementosMovimiento(query string) (elementos []*models.ElementosMovimiento, outputError map[string]interface{}) {
```